jsx/App/locale/TranslatableText.jsx

```diff
@@ -9,6 +9,11 @@ export const TranslatableText = ({ dictionary }) => {
   const { locale } = useContext(LocaleContext);
   if (dictionary[locale] === undefined) {
     console.error(`[TranslatableText] No translation for ${locale} in provided dictionary`);
```

Comment:
It sounds like this case is expected, in some cases, so we probably shouldn't log an error here. It will probably be alarming/annoying to folks developing. Might be better to only log an error if `undefined` is returned for whatever reason.

Reply:
Good point